pkgs/by-name/ge/geopard/package.nix

```
@@ -31,8 +31,9 @@ stdenv.mkDerivation rec {
     hash = "sha256-elHxtFEGkdhEPHxuJtcMYwWnvo6vDaHiOyN51EOzym0=";
   };
 
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
```

```suggestion
    inherit pname version src;
```

To give the tarball a meaningful name.

---

thanks ! good to know